File: python/problem27.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        length = len(nums);
        result = 0;
        for i in range(length):
            if nums[i] == val:
                continue;
            else:
                nums[result] = nums[i];
                result += 1;
        return result;

File: Java/problem27.java
public class Solution
{
    public int removeElement(int[] nums, int val) 
    {
        int length = nums.length;
        int result = 0;
        for(int i = 0; i < length; ++i)
        {
            if(nums[i] == val)
                continue;
            else
            {
                nums[result] = nums[i];
                ++result;
            }
        }
        return result;
    }
}